internal/scorecard/kubeclient.go

```diff
@@ -47,3 +49,50 @@ func GetKubeClient(kubeconfig string) (client kubernetes.Interface, err error) {
 
 	return clientset, err
 }
+
+// GetKubeNamespace returns the kubernetes namespace to use
+// for scorecard pod creation
+// the order of how the namespace is determined is as follows:
+// - a namespace command line argument
+// - a namespace determined from the kubeconfig file
+// - the kubeconfig file is determined in the following order:
+//   - from the kubeconfig flag if set
+//   - from the KUBECONFIG env var if set
+//   - from the $HOME/.kube/config path if exists
+//   - returns 'default' as the namespace if not set in the kubeconfig
+func GetKubeNamespace(kubeconfigPath, namespace string) string {
+
+	if namespace != "" {
+		return namespace
+	}
+
+	rules := clientcmd.ClientConfigLoadingRules{
```

Comment:
I recommend using [`NewDefaultClientConfigLoadingRules`](https://godoc.org/k8s.io/client-go/tools/clientcmd#NewDefaultClientConfigLoadingRules`) to get loading rules to avoid hard-coded path lookup using `getConfigPath`.

Reply:
good catch, changed to that API call.